package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jupyterlab-h5web",
-  "version": "8.0.0",
+  "version": "8.1.0",
```

Comment:
You can leave the version number unchanged. 

We have a dedicated release process that update it automatically.